app/models/pulitzer/content_element.rb

```diff
@@ -34,6 +45,37 @@ def empty_body?
       image_type? ? !image? : body.blank?
     end
 
+    def clone_me
+      clone_attrs = self.attributes.dup
+      clone_attrs.delete 'id'
+      clone_attrs.delete 'version_id'
+
+      type_clone_method = 'clone_' + type.to_s
+      if respond_to? type_clone_method
+        my_clone = send type_clone_method, clone_attrs
```

Comment:
A good rule of thumb: if I can call `send` without a left side, then it's OK because I'm calling it on `self`. If I have to call it on an object like so `some_object.public_send` then `public_send` is probably the way to go.